lib/bdev/bdev.c

```diff
@@ -857,6 +859,36 @@ spdk_bdev_reset(struct spdk_bdev *bdev, enum spdk_bdev_reset_type reset_type,
 	return rc;
 }
 
+struct spdk_bdev_io *
```

Comment:
It works whether we call submit_request with ch = NULL or = bdev_io->ch->channel since we don't use it. But we have used the reset code as a model for a command that does not need an I/O channel, so we prefer to set it to NULL to be consistent with that model. Any thoughts?